src/10-ipscend-update/index.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2016-02-22
+id: 10-ipscend-update
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "10-ipscend-update", link: "./" }
+tags: publishing, web, app, publish
+title: ipscend update, now compatible with IPFS 0.3.11
+author: David Dias
+collection: posts
+---
+
+[![](!img/ipscend.png)](https://github.com/diasdavid/ipscend)
+
+> Web Application publishing, simple and distributed with IPFS
+
+
+** The case for this tool 
+
+# Using ipscend
+
+- [ ] asciinema / video
+
+# Users friendly
+
+- [ ] explain naming problem
+- [ ] explain how IPFS public gateways can be the proxies for users seemlessly (with graphs!)
```

Comment:
Doesn't it have a different purpose? Here would be explaining how we can get DNS names to load IPFS hashes through using IPFS public gateways as a proxy